### PR TITLE
8.13 update custom documentation

### PR DESCRIPTION
**Pull request by intxgo**
## Change Summary

Adds document fields introduced in 8.13 and custom documentation for them.

Changes made using documents data from testing automation, where custom documentation was generated with `python3 -m endpoint.tools.update_custom_documentation`

Unfortunately there's lots of changes to "short description" fields which are unrelated to the PR. As far as I know, short description lines should not exceed 120 characters. It seems that `make clean all` reformatted the documentation to comply with the standard.

### Sample values

Sample document:

```json


```

### For mapping changes:

- [x] I ran `make` after making the schema changes, and committed all changes
- [ ] If these field(s) are "exception"-able, I made a companion PR to Kibana adding it (see [Readme](https://github.com/elastic/endpoint-package#exceptionable))
- [ ] If this is a `metadata` change, I also updated both transform destination schemas to match

### For Transform changes:

- [ ] The new transform successfully starts in Kibana
- [ ] The corresponding transform destination schema was updated if necessary

### CODE REVIEW

**Review by pzl — APPROVED** [review]
weird on the reformatting. approved tho

**Review by nfritts — APPROVED** [review]

**Review by ferullo — COMMENTED** [review]
Changes in `custom_documentation/src` LGTM. I didn't review the rest since it's all auto generated and you didn't change the tooling.

~~@pzl since this incidentally updated files outside `custom_documentation/` can you 👍 it too?~~ you already did 😄

**Comment by pzl** [comment]
It looks like CI is disagreeing with the formatting change.

In a rarity, if only  `custom_documentation` is changing, then `make` is unnecessary here, since the custom docs folder are not included in the shipped package contents.

the `make clean all` certainly remade things that CI's `make` is doing in a different order.

You can safely, manually,  back-out all the changes under `package/endpoint` so this PR is just the stuff under custom_documentation and I expect CI will pass

**Comment by intxgo** [comment]
@pzl thanks for the help, actually it was my fault, I also fixed some randomly spot description and forgot about it, that's why it's failing https://github.com/elastic/endpoint-package/pull/496/commits/6bfa4ebbdf30194d79bb98e721c7ddea5ad39555

I'll revert it for now, but perhaps it should be fixed elsewhere

**Comment by intxgo** [comment]
Ok, reverting the other changes was not a good solution, as now the build complains about descriptions exceeding 120 chars per line.

However I've found what's the culprit, something else is returning `exit 1` during `make clean all`

```
2024/04/22 11:11:08 generating docs failed: rendering custom documentation failed (packageName: endpoint): failed to render custom_documentation/src/endpoint/data_stream/volume_device/windows/windows_volume_device_mount.yaml: opening custom_documentation/doc/endpoint/volume_device/windows/windows_volume_device_mount.md for writing failed, does the directory exist?: open custom_documentation/doc/endpoint/volume_device/windows/windows_volume_device_mount.md: no such file or directory
exit status 1
make: *** [Makefile:83: package/endpoint/docs/README.md] Error 1
```

File: custom_documentation/doc/endpoint/process/windows/windows_process_exit.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ This event is generated when a process exits.
 | process.Ext.code_signature.status |
 | process.Ext.code_signature.subject_name |
 | process.Ext.code_signature.trusted |
+| process.Ext.hidden.behaviors.api |
```
Comment (jdu2600) [inline custom_documentation/doc/endpoint/process/windows/windows_process_exit.md:51]: This is not a valid field.

File: custom_documentation/doc/endpoint/process/windows/windows_process_create.md [hunk 2]
```diff
@@ -49,6 +49,7 @@ This event is generated when a process is created.
 | process.Ext.code_signature.status |
 | process.Ext.code_signature.subject_name |
 | process.Ext.code_signature.trusted |
+| process.Ext.created_suspended |
```
Comment (jdu2600) [inline custom_documentation/doc/endpoint/process/windows/windows_process_create.md:52]: This field is 8.14+ and should not be included here.

**Pull request merged**